pkg/manager/member/tikv_member_manager.go

```diff
@@ -209,6 +215,13 @@ func (tkmm *tikvMemberManager) syncStatefulSetForTidbCluster(tc *v1alpha1.TidbCl
 		return err
 	}
 
+	if tkmm.webhookEnabled {
+		err := tkmm.tikvRestarter.Sync(tc, v1alpha1.TiKVMemberType)
+		if err != nil {
+			return err
+		}
+	}
```

Comment:
IIUC, restarting will have to wait `0.5 * ResyncPeroid` in average if there is no event triggers reconciling, and a Pod will not get restarted if there is anything wrong in the former reconciling procedures.

I think we should add a new controller and make it orthogonal with tdb-cluster controller